modules/vidoomyBidAdapter.js

```diff
@@ -185,91 +194,32 @@ const interpretResponse = (serverResponse, bidRequest) => {
   }
 };
 
+function getUserSyncs (syncOptions, responses, gdprConsent, uspConsent) {
```

Comment:
You are not using `uspConsent` anywhere.  

You must include `uspConsent` string in your cookie sync urls.